eo-maven-plugin/src/main/java/org/eolang/maven/OyRemote.java

```diff
@@ -81,4 +75,12 @@ public Input get(final String name) throws MalformedURLException {
         return new InputOf(url);
     }
 
+    /**
+     * Accessor.
+     * @return Template
+     */
+    public String getTemplate() {
```

Comment:
@yasamprom getters is a bad practice: https://www.yegor256.com/2014/09/16/getters-and-setters-are-evil.html